libsignal-service/build.rs

```diff
@@ -28,5 +28,9 @@ fn main() {
         })
         .collect();
 
-    prost_build::compile_protos(&input, &[protobuf]).unwrap();
+    let mut prost_build = prost_build::Config::new();
+    prost_build
+        .default_package_filename("signal")
```

Comment:
Does this break compat?

Reply:
Quite the opposite, this is there to keep compatibility 🪨